Dockerfile

```diff
@@ -34,8 +34,6 @@ ENV BIGCHAINDB_SERVER_BIND 0.0.0.0:9984
 # but maybe our Docker or Docker Compose stuff does?
 # ENV BIGCHAINDB_API_ENDPOINT http://bigchaindb:9984/api/v1
 
-ENTRYPOINT ["bigchaindb", "--dev-start-rethinkdb", "--dev-allow-temp-keypair"]
+ENTRYPOINT ["bigchaindb"]
```

Comment:
As I understand, `--dev-allow-temp-keypair` flag is used to create a [temp keypair](https://github.com/bigchaindb/bigchaindb/blob/e6d99d74c022e6ca7954786b7db106f9fe36d98a/bigchaindb/commands/bigchain.py#L183).
If we go into production with this, we should be able to set a keypair ourselves rather than let bigchaindb generate it.
We also need to do key management and periodic recycling of the keys.
I have removed it as a mandatory `entrypoint` option, and let devs specify it explicitly during testing, as is documented below.